docs/providers/aws/events/schedule.md

```diff
@@ -36,7 +36,9 @@ functions:
       - schedule:
           rate: rate(10 minutes)
           enabled: false
+          input: '{"key": "value"}'
```

Comment:
It would be nice to support here also yaml syntax to define the input as an object. It would come much more handy for more complex objects.

In such case it would be nice to also show this example, eg:

```yaml
- schedule:
  rate: rate(10 minutes)
  enabled: false
  input:
     key1: value1
     key2: value2
     stageParams:
       stage: dev
```